### ISSUE TITLE
3.47.2 version breaks auto-approve CLI

### ISSUE THREAD

**Issue by fbarrerafalabella** [pulumi/pulumi#11451]
### What happened?

with this new release I cannot perform the following command on my CI/CD pipeline
```
pulumi up -s <stack> --yes
```
I tells me the following 
```
error: resource violates plan: properties changed
```
When I manually execute 
```
pulumi up -s <stack> 
```
and then I select yes it executes as normal

### Steps to reproduce

Execute pulumi up -s <stack> --yes

### Expected Behavior

Apply the changes without any problem

### Actual Behavior

shows this message
```
error: resource violates plan: properties changed
```

### Output of `pulumi about`
```
pulumi about
CLI
Version      3.47.2
Go Version   go1.19.3
Go Compiler  gc

Plugins
NAME    VERSION
nodejs  unknown

Host
OS       darwin
Version  13.0.1
Arch     x86_64

This project is written in nodejs: executable='/usr/local/bin/node' version='v16.14.2'

Backend
Name           pulumi.com
URL            https://app.pulumi.com/fbarrerafalabella
User           fbarrerafalabella
Organizations  fbarrerafalabella, LinioIT
```

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

**Comment by fbarrerafalabella** [comment]
when I do a 
```
pulumi up -s LinioIT/staging --yes --skip-preview
```
it works, I think the preview is what is causing the problem

**Comment by ivan-amity** [comment]
I've got this issue and this can pass by adding the `-f` flag. 

`pulumi up -r -s <stack_name> -y -f`

**Comment by fbarrerafalabella** [comment]
Yes, but it should work showing the preview as before doesnt it?

**Comment by justinvp** [comment]
Duplicate of https://github.com/pulumi/pulumi/issues/11444, which has been fixed in v3.48.0.
